backburner/values.yaml

```diff
@@ -12,6 +12,21 @@ global:
   envFromSecret:
     - name: secret-env
   affinity: {}
+#   nodeSelector:
```

Comment:
I suggest modifying the configuration for `nodeSelector`, `tolerations`, and `topologySpreadConstraints` to the following format, which:

1. Provides clearer example values and comment format
2. Makes it easier for future maintainers to understand and modify
3. Maintains consistency and readability in the configuration structure

This format maintains the default empty configuration while providing reference examples for actual implementation.
```
nodeSelector: {}
#   nodeSelector:
#     labelName: nodegroup
#     labelValue: "sl1.0businessplatform-businessplatform"
tolerations: []
#   tolerations:
#   - key: "sl1.0businessplatform-businessplatform"
#     operator: "Equal"
#     value: "true"
#     effect: "NoSchedule"
topologySpreadConstraints: []
#   topologySpreadConstraints:
#     - labelSelector:
#         matchLabels:
#           app: app-name
#       maxSkew: 1
#       topologyKey: topology.kubernetes.io/zone
#       whenUnsatisfiable: ScheduleAnyway
```